tests/test_exports.json

```diff
@@ -141,7 +141,7 @@
 
     {
         "label": "Entity with data-*",
-        "output": "<p><a data-id=\"42\" data-mutability=\"mutable\" href=\"/\" title=\"hi\"><em>a</em></a></p>",
+        "output": "<p><a data-=\"no\" data-false=\"bad\" data-id=\"42\" data-mutability=\"mutable\" extra=\"foo\" href=\"/\" title=\"hi\"><em>a</em></a></p>",
```

Comment:
I think we are testing the behavior of `html5lib` here:

- lowercase attributes
- alphabetical order
- ... and potentially other things.

In the [BeautifulSoup doc](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser), it is classified as "extremely lenient", which is illustrated here. I don't think we have to care too much for now.